app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -51,6 +58,87 @@ angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap'])
     $interval.cancel(promise);
   });
 
+
+  var refreshProvider = function(d) {
+    $timeout(function(){
```

Comment:
Indeed that's what I meant (well, I would have put it around all d3 event handlers, but this is the only relevant place, so yeah) .. But ng-show doesn't expect the element to be css-hidden, so if you remove that `display:none` in css, it should work now.